packages/components/inputs/search-text-input/src/search-text-input.spec.js

```diff
@@ -0,0 +1,145 @@
+import { render, fireEvent, screen } from '../../../../../test/test-utils';
+import SearchTextInput from './search-text-input';
+
+const baseProps = {
+  value: '',
+  onChange: () => {},
+  onSubmit: () => {},
+  onReset: () => {},
+};
+
+describe('SearchTextInput.isEmpty', () => {
+  describe('when called with an empty value', () => {
+    it('should return true', () => {
+      expect(SearchTextInput.isEmpty('')).toBe(true);
+      expect(SearchTextInput.isEmpty(' ')).toBe(true);
+    });
+  });
+  describe('when called with a filled value', () => {
+    it('should return false', () => {
+      expect(SearchTextInput.isEmpty('a')).toBe(false);
+      expect(SearchTextInput.isEmpty(' a ')).toBe(false);
+    });
+  });
+});
+
+describe('SearchTextInput', () => {
+  it('should forward data-attributes', () => {
+    const { container } = render(
+      <SearchTextInput {...baseProps} data-foo="bar" />
+    );
+    expect(container.querySelector('input')).toHaveAttribute('data-foo', 'bar');
+  });
+
+  it('should render a number input', () => {
+    const { container } = render(<SearchTextInput {...baseProps} />);
+    expect(container.querySelector('input')).toHaveAttribute('type', 'text');
+  });
```

Comment:
Not sure if the title of the test is accurate.
It seems we're just trying to render the component with an empty value and checking there's a `input` element rendered.

Reply:
Updated [f6ff5c6](https://github.com/commercetools/ui-kit/pull/2411/commits/f6ff5c6ad2b798eac4bc285afba163f835a36f8a)